pillarbox-player/src/main/java/ch/srgssr/pillarbox/player/notification/PillarboxMediaDescriptionAdapter.kt

```diff
@@ -67,7 +69,9 @@ class PillarboxMediaDescriptionAdapter(
                 val imageUri = player.mediaMetadata.artworkUri!!
                 val artworkBitmap = bitmapCache.get(imageUri)
                 if (artworkBitmap == null) {
-                    loadBitmapFromUri(imageUri, callback)
+                    MainScope().launch {
```

Comment:
Here I would:
- Extract `MainScope` in a field, to not recreate it each time.
- Pass a `CoroutineContext` to `launch`, so you don't need the `withContext(Dispatchers.IO)` wrapper in `loadBitmapFromUri()`.
```suggestion
                    coroutineScope.launch(Dispatchers.IO) {
```